### PR TITLE
PD-1487 / 25.04 / replace minio plugin and migration content with EoL statement (master)

### PR DESCRIPTION
**Pull request by Mrt134**
Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

**Review by chrismillsapsix — DISMISSED** [review]

**Comment by Mrt134** [comment]
Note that I will manually backport most of these changes to the 13.0 and 13.3 branches when this is merged.

File: content/Solutions/MinioCOREtoSCALE.md [hunk 1]
```diff
@@ -9,67 +9,14 @@ tags:
 - apps
 ---
 
-TrueNAS CORE users utilizing the MinIO plugin can migrate to TrueNAS SCALE and keep their existing MinIO buckets and data.
-This process does not transfer users, groups, access keys, and all other MinIO settings.
-They must be manually recreated on the new SCALE app deployment.
+{{< hint type="warning" title="Minio Plugin End of Life" >}}
+Due to the interaction between the TrueNAS, Minio, and FreeBSD software lifecycles, the Minio plugin does not generally deploy in TrueNAS 13.0 nor is full data migration of MinIO data from end of life (EoL) deployments of MinIO possible.
+Users are encourage to pursue a manual backup and restore strategy to move data into the fully maintained and updatable MinIO applications available with TrueNAS 24.04 and later.
```
Comment (DjP-iX) [inline content/Solutions/MinioCOREtoSCALE.md:14]:
```suggestion
Users are encouraged to pursue a manual backup and restore strategy. 
Sidegrade to TrueNAS 24.04 or newer to move data into the available, fully maintained, and updatable MinIO [applications](https://www.truenas.com/docs/truenasapps/).
```

File: content/Solutions/MinioCOREtoSCALE.md [hunk 2]
```diff
@@ -9,67 +9,14 @@ tags:
...
-   Adjust permissions to ensure the **apps** user has full control and select **Apply permissions recursively**.
-   Click **Save Access Control List**.
-
-   {{< trueimage src="/images/Solutions/MinIODatasetPermissions.png" alt="MinIO Dataset Permissions" id="MinIO Dataset Permissions" >}}
-
-3. Go to **Apps**.
-   Click **Discover Apps**, then either begin typing MinIO into the search field or scroll down to locate either the **charts** (community) or **enterprise** version of the [**MinIO**](https://www.truenas.com/docs/scale/scaletutorials/apps/communityapps/minioapp/) app widget.
-  
-   Community users can install the Enterprise version of the app by following the procedure in [Adding MinIO Enterprise App](https://www.truenas.com/docs/scale/scaletutorials/apps/enterpriseapps/minio/#adding-minio-enterprise-app).
+Enterprise customers are encouraged to contact iX Support to discuss options for guided data migration from the EoL S3 service or Minio Plugin in TrueNAS 13.0 and into supported MinIO Enterprise applications available in TrueNAS 24.04, 24.10, and newer releases.
```
Comment (DjP-iX) [inline content/Solutions/MinioCOREtoSCALE.md:16]:
```suggestion
Enterprise customers are encouraged to contact iX Support to discuss options for guided data migration from the EoL S3 service or Minio Plugin in TrueNAS 13.0 to the supported [MinIO Enterprise](https://www.truenas.com/docs/truenasapps/enterpriseapps/minio/) applications available in TrueNAS 24.04 and newer releases.
```

File: static/includes/S3Deprecation.md [hunk 3]
```diff
@@ -1,16 +1,18 @@
 &NewLine;
 
 {{< hint type=important title="Deprecation Notice" >}}
-Due to security vulnerabilities and maintainability issues, the S3 service is deprecated in TrueNAS 13.0 and removed in CORE 13.3.
+Due to security vulnerabilities and maintainability issues, the S3 service is deprecated in TrueNAS 13.0 and removed in TrueNAS 24.04, 24.10, and newer versions.
```
Comment (DjP-iX) [inline static/includes/S3Deprecation.md:4]:
```suggestion
Due to security vulnerabilities and maintainability issues, the S3 service is deprecated in TrueNAS 13.0 and removed in TrueNAS 22.12 and newer versions.
```

**Review by DjP-iX — CHANGES_REQUESTED** [review]

**Review by micjohnson777 — APPROVED** [review]

**Pull request merged**